packages/components/src/utils/colors.values.js

```diff
@@ -0,0 +1,115 @@
+/**
```

Comment:
> This makes me wonder, whether there is tooling which would allow to generates SCSS file from this JS file :)

Yup! Maybe not out of the box, but possible. I built a system like that here:
https://github.com/helpscout/colorway

Reply:
cc @porada in case you'd like to help out here. ;-)